src/toil/utils/toilStatus.py

```diff
@@ -94,7 +96,7 @@ def printAggregateJobStats(self, properties, childNumber):
                              lf("READY_TO_RUN"), lf("IS_ZOMBIE"),
                              lf("HAS_SERVICES"), lf("IS_SERVICE"))))
 
-    def report_on_jobs(self):
+    def report_on_jobs(self) -> Dict[str, Any]:
```

Comment:
I was trying to use Dict[str, List[Job]] as the type but mypy was throwing a warning unless Any was used. I'm going to try seeing if I can make the typing more specific.

Reply:
Sounds good!

Reply:
Some of the dict values are `List[Job]`, but `'properties'` is `Set[str]`, so the whole thing comes out to I think `Dict[str, Union[List[Job], Set[str]]]`, which isn't necessarily much more useful than `Dict[str, Any]` was.